app/scripts/directives/nav.js

```diff
@@ -273,10 +294,54 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        contextSelector.append($('<option value="1">Service Catalog</option>'));
+        contextSelector.append($('<option value="2">Application Console</option>'));
+        if (window.OPENSHIFT_CONSTANTS.TECTONIC_URL) {
```

Comment:
Works for me for now. The goal is to have the tectonic console running on the same domain, so we might be able to put it in a well-known place like `/cluster-console` (or wherever) with no config needed.